packages/nyc-config-hook-run-in-this-context/package.json

```diff
@@ -32,6 +32,11 @@
   "dependencies": {
     "semver": "^6.0.0"
   },
+  "devDependencies": {
+    "chai": "^4.2.0",
+    "mocha": "^6.1.4",
+    "nyc": "^14.1.0"
```

Comment:
just a quick question /curiousity, remind me how lerna's hoisting works again? it just worries me having so many copies of these lines with nyc, mocha, and chai that could drift...

Reply:
All of these packages are deduplicated to the root node_modules but a symlink is created for the binaries.  So `packages/nyc-config-hook-run-in-this-context/node_modules/.bin/nyc` will be a symlink to the binary in the root node_modules.

Any difference in raw range will cause warning on `npm install`, so if one package depends on `^14.1.0` and another depends on `^14.1.1` it will warn.  These are all managed by greenkeeper anyways so for example all packages will always be upgraded to nyc ^15.0.0 at the same time.